app/src/main/java/mozilla/lockbox/presenter/FingerprintDialogPresenter.kt

```diff
@@ -37,7 +35,7 @@ class FingerprintDialogPresenter(
             .addTo(compositeDisposable)
 
         view.authCallback
-            .subscribe { dispatcher.dispatch(OnAuthentication(it)) }
+            .subscribe { dispatcher.dispatch(it) }
```

Comment:
oo, we can move to using the `::` syntax here now, so this can read `.subscribe(dispatcher::dispatch)` !!